internal/declarative/v2/test/v1/declarative_test.go

```diff
@@ -189,9 +414,11 @@ func StartDeclarativeReconcilerForRun(
 	go func() {
 		Expect(mgr.Start(ctx)).To(Succeed(), "failed to run manager")
 	}()
+	return reconciler.(*Reconciler)
 }
 
-func StatusOnCluster(g Gomega, ctx context.Context, key client.ObjectKey) Status { //nolint:revive
+func StatusOnCluster(g Gomega, ctx context.Context, key client.ObjectKey,
```

Comment:
```suggestion
func StatusOnCluster(ctx context.Context, key client.ObjectKey,
```
We can remove the Gomega parameter here. This enables us to remove the `//nolint:revive` exception